Formula/lnav.rb

```diff
@@ -1,7 +1,7 @@
 class Lnav < Formula
   desc "Curses-based tool for viewing and analyzing log files"
   # lnav.org has an SSL issue: https://github.com/tstack/lnav/issues/401
-  homepage "https://github.com/tstack/lnav"
+  homepage "https://lnav.org"
   url "https://github.com/tstack/lnav/releases/download/v0.8.5/lnav-0.8.5.tar.gz"
   sha256 "bb809bc8198d8f7395f3de76efdc1a08a5c2c97dc693040faee38802c38945de"
 
```

Comment:
please add a revision here like:
```suggestion
revision 1

```